eng/common/Maestro-PublishBuildToMaestro.yml

```diff
@@ -103,15 +93,27 @@ steps:
       filePath: eng\common\MaestroPostRequest.ps1
       arguments: -url '$(MaestroUri)' -Token '$(MaestroToken)' -api '/api/builds' -jsonBodyPath '$(Build.SourcesDirectory)\eng\common\maestro-build.json'
  
-  - ${{ if eq(parameters.TriggerSubscrption, 'true') }}:
+  - ${{ if eq(parameters.TriggerSubscription, 'true') }}:
     - task: powershell@2
       displayName: 'Publish Build to Default Channel'
       inputs:
         targetType: 'inline'
         script: |
+          . .\eng\common\MaestroHelpers.ps1
```

Comment:
>.  [](http://example.com/codeflow?start=10&length=2)

what is this?

Reply:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_scopes?view=powershell-5.1#using-dot-source-notation-with-scope